src/components/Class/index.js

```diff
@@ -337,14 +342,22 @@ function Class({ classToEdit, indicator }) {
                       setFormField("cohort", TYPE);
                     }}
                     value={formFieldsState[TYPE]}
-                    id="type3"
+                    id="type1"
                     checked={
                       formFieldsState.type === "cohort" ? "checked" : false
                     }
+                    // disabled={formFieldsState[TYPE] === "cohort" ? true : false}
+                    disabled={
+                      isEditMode
+                        ? formFieldsState[TYPE] === "cohort"
+                          ? false
+                          : true
+                        : false
+                    }
```

Comment:
In order for this expression to evaluate to `true`, `isEditMode` needs to be `true` and `formFieldsState[TYPE] === "cohort"` needs to be `false` so you can write this as follows:
```suggestion
                    disabled={isEditMode && formFieldsState[TYPE] !== "cohort"}
```